platform/platform-app/src/main/resources/features.xml

```diff
@@ -988,6 +989,12 @@
         <bundle dependency="true">mvn:com.google.guava/guava/${guava.version}</bundle>
     </feature>
 
+    <feature name="commons" version="${project.version}" install="manual" hidden="true">
+        <bundle dependency="true">mvn:commons-io/commons-io/${commons-io.version}</bundle>
+        <bundle dependency="true">mvn:commons-lang/commons-lang/${commons-lang.version}</bundle>
+        <bundle dependency="true">mvn:commons-collections/commons-collections/${commons-collections.version}</bundle>
+    </feature>
```

Comment:
@stustison or @pklinef please take note what I've done to the platform feature to account for adding dependencies to platform util - let me know 👍  or 👎